core/modules/css/KvInterfaceImplMySql.cc

```diff
@@ -568,9 +568,11 @@ std::string KvInterfaceImplMySql::dumpKV() {
         if (row[1].first == nullptr or *row[1].first == '\0') {
             result += "\\N";
         } else {
+/*
             if (strchr(row[1].first, '\n') != nullptr) {
                 throw CssError("KvInterfaceImplMySql::dumpKV - value contains newline");
             }
+*/
```

Comment:
I don't remember any outstanding tickets for qserv-admin. We should improve it of course to support all the features that we need. We don't care about RESTORE for now, maybe we should wait until there is a use case for that.